pkgs/by-name/do/dove/package.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  fetchzip,
+  stdenvNoCC,
+  nix-update-script,
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "dove";
+  version = "2025.02.01.1";
+
+  src = fetchzip {
```

Comment:
Use fetchFromGitea.

Reply:
this will fetch the whole repo with all the build scripts and more, whereas I'm only interested in the few files in the zip archive. Is that ok?

Reply:
It also fetches the zip.

Reply:
the current code does not fetch the zip of the repo, it only fetch a zip inside the repo that contains only necessary files.